### PR TITLE
VxCentralScan desktop theme

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

Converts VxCentralScan to use the desktop theme. Major changes:
- Adding a persistent left nav bar
- Moving the "Scan New Batch" and "Export CVRs" buttons out of the header and into the main content
- Changing the test ballot mode banner to a smaller callout
- Reorganizing the system admin screen

## Demo Video or Screenshot
Election manager - Configure/unconfigure
https://github.com/votingworks/vxsuite/assets/530106/146b778c-70f4-4852-8b95-e578dd080090

Election manager - Scanning ballots and saving CVRs
https://github.com/votingworks/vxsuite/assets/530106/d2b602f9-72fb-42dc-b9da-afe8353f7ace

Election manager - Adjudication
Screenshot-VxCentralScan-2023-11-21T21:55:41 802Z

System admin
Screenshot 2023-11-21 at 1 19 15 PM

Test ballot mode

Screenshot-VxCentralScan-2023-11-21T21:21:58 827Z

## Testing Plan
- Manual testing
- (TODO) Update automated tests

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

**Comment by jonahkagan** [comment]
@adghayes @mattroe Ready for design feedback while I work on tests.

**Comment by mattroe** [comment]
@jonahkagan can you throw up a screenshot or video of the settings screen?

**Comment by jonahkagan** [comment]
> @jonahkagan can you throw up a screenshot or video of the settings screen?

It's in the configure/unconfigure video above

**Comment by mattroe** [comment]
> > @jonahkagan can you throw up a screenshot or video of the settings screen?
> 
> It's in the configure/unconfigure video above

Oh missed that, thanks!

**Comment by mattroe** [comment]
New design system looks great work. Overall seems good to go.

This change is making me realize that there are a few minor design/copy inconsistencies that we should address. I'll document these holistically across apps so we can do a quick clean up pass before this release.

**Comment by jonahkagan** [comment]
> This change is making me realize that there are a few minor design/copy inconsistencies that we should address.

Yes, I noticed some similar inconsistencies.

**Review by adghayes — COMMENTED** [review]
Design & design choices LGTM.

File: libs/ui/src/left_nav.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { Icons } from './icons';
 export const LeftNav = styled.nav`
   background: ${(p) => p.theme.colors.inverseBackground};
   padding: 1rem;
-  width: 14rem;
+  width: 15rem;
```
[the remark this answers is not in the document]
Comment (jonahkagan) [reply libs/ui/src/left_nav.tsx:12]: Yes, it does

**Review by adghayes — APPROVED** [review]

**Pull request merged**